content/swiftui/concepts/tables/tables.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: 'Tables' 
+Description: 'A container that displays data in rows organized into one or more columns and optionally allows the selection of one or more items.'
+Subjects: 
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'iOS'
+  - 'SwiftUI'
+  - 'Tables'
+CatalogContent:
+  - 'learn-ios-course'
+  - 'paths/ios-developer'
+---
+
+**`Table`** in SwiftUI is a container that enables the creation and presentation of data collections in a tabular format, organized into columns and rows. It is a valuable tool for displaying datasets that have a structured, grid-like layout. When the number of rows exceeds the available vertical space, the `Table` component will automatically enable vertical scrolling to ensure all data is accessible. Moreover, on macOS, horizontal scrolling becomes available when the number of columns exceeds the width of the view.
+
```

Comment:
Before any examples there should be a syntax section to demonstrate how the structure is implemented generically with pseudo code. For example - 
https://www.codecademy.com/resources/docs/swiftui/views/button